example/lib/main.dart

```diff
@@ -170,6 +173,42 @@ class _MyAppState extends State<MyApp> {
       print("Failed to open document: '${e.message}'.");
     }
   }
+
+  void showFormDocumentExample() async {
+    try {
+      final ByteData bytes =
+          await DefaultAssetBundle.of(context).load(_formPath);
+      final Uint8List list = bytes.buffer.asUint8List();
+
+      final tempDir = await getTemporaryDirectory();
+      final tempDocumentPath = '${tempDir.path}/$_formPath';
+
+      final file = await File(tempDocumentPath).create(recursive: true);
+      file.writeAsBytesSync(list);
+
+      Pspdfkit.present(tempDocumentPath);
+
+      Pspdfkit.setFormFieldValue("Lastname", "Name_Last");
+      Pspdfkit.setFormFieldValue("0123456789", "Telephone_Home");
+      Pspdfkit.setFormFieldValue("City", "City");
+      Pspdfkit.setFormFieldValue("selected", "Sex.0");
+      Pspdfkit.setFormFieldValue("deselected", "Sex.1");
+      Pspdfkit.setFormFieldValue("selected", "HIGH SCHOOL DIPLOMA");
+
+      String lastName;
+      // Platform messages may fail, so we use a try/catch PlatformException.
+      try {
+        lastName = (await Pspdfkit.getFormFieldValue("Name_Last")) as String;
+      } on PlatformException {
+        lastName = 'Failed to get last name.';
```

Comment:
It looks a bit odd that the last name is the error message. Perhaps change the variable name to `error` or `errorMessage` and only print that, instead of printing the last name all the time.